PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -252,51 +237,41 @@ def miniAOD_customizeCommon(process):
     process.patJetsPuppi.userData.userFloats.src = cms.VInputTag(cms.InputTag(""))
     process.patJetsPuppi.jetChargeSource = cms.InputTag("patJetPuppiCharge")
 
-    process.selectedPatJetsPuppi.cut = cms.string("pt > 20")
+    process.selectedPatJetsPuppi.cut = cms.string("pt > 15")
 
     process.load('PhysicsTools.PatAlgos.slimming.slimmedJets_cfi')
     process.slimmedJetsPuppi = process.slimmedJets.clone()
     process.slimmedJetsPuppi.src = cms.InputTag("selectedPatJetsPuppi")    
     process.slimmedJetsPuppi.packedPFCandidates = cms.InputTag("packedPFCandidates")
 
-    ## puppi met
-    process.load('RecoMET.METProducers.PFMET_cfi')
-    process.puppiForMET = cms.EDProducer("CandViewMerger",
-        src = cms.VInputTag( "pfLeptonsPUPPET", "puppiNoLep")
-    ) 
-    process.pfMetPuppi = process.pfMet.clone()
-    process.pfMetPuppi.src = cms.InputTag("puppiForMET")
-    process.pfMetPuppi.alias = cms.string('pfMetPuppi')
```

Comment:
Is this removal intentional?
The result is that pfMetPuppi will come out with an alias pfMet.
I could speculate on reasons why the pfMet alias may be desired to point to pfMetPuppi.
Please confirm.